MIGRATING-0.6-0.7.md

```diff
@@ -113,6 +116,37 @@ TODO: Improve this section before release.
 - New fonts with `ascii` and `latin1` glyph subsets
 - `MonoFont` is now a struct instead of a trait
 
+The list of fonts has changed to the following:
+
+All fonts are provided in `embedded_graphics::mono_font::[ascii|latin1]::[font name]`
+
+- `Font4x6`
+- `Font5x7`
+- `Font5x8`
+- `Font6x10`
+- `Font6x12`
+- `Font6x13`
+  - Also available in **bold** (`Font6x13Bold`) and _italic_ (`Font6x13Italic`)
+- `Font6x9`
+- `Font7x13`
+  - Also available in **bold** (`Font7x13Bold`) and _italic_ (`Font7x13Italic`)
+- `Font7x14`
+  - Also available in **bold** (`Font7x14`)
+- `Font8x13`
+  - Also available in **bold** (`Font8x13Bold`) and _italic_ (`Font8x13Italic`)
+- `Font9x15`
+  - Also available in **bold** (`Font9x15Bold`)
+- `Font9x18`
+  - Also available in **bold** (`Font9x18Bold`)
+- `Font10x20`
+
+Two character sets are now provided for each font. The `ascii` set can be use to reduce memory consumption by the font data, at the cost of losing extra characters.
```

Comment:
I think I would drop the "losing" part, instead of stating the pros of each, something along these lines:

```suggestion
Two character sets are now provided for each font. The `ascii` set has a reduced memory footprint, while `latin1` contains more characters.
```